mito-ai/src/Extensions/AiChat/ChatMessage/ChatInput.tsx

```diff
@@ -142,55 +171,92 @@ const ChatInput: React.FC<ChatInputProps> = ({
+                render the dropdown, it is relative to the text area instead of the entire 
+                div. We do this so that the dropdown sits on top of (ie: covering) the code 
+                preview instead of sitting higher up the taskpane.
+            */}
+            <div style={{ position: 'relative' }}>
+                <textarea
+                    ref={textAreaRef}
+                    className={classNames("message", "message-user", 'chat-input')}
+                    placeholder={placeholder}
+                    value={input}
+                    onChange={handleInputChange}
+                    onKeyDown={(e) => {
+                        // If dropdown is visible, only handle escape to close it
+                        if (isDropdownVisible) {
+                            if (e.key === 'Escape') {
+                                e.preventDefault();
+                                setDropdownVisible(false);
+                            }
+                            return;
+                        }
+
+                        // Enter key sends the message, but we still want to allow 
+                        // shift + enter to add a new line.
+                        if (e.key === 'Enter' && !e.shiftKey) {
                             e.preventDefault();
-                            setDropdownVisible(false);
+                            adjustHeight(true)
+                            onSave(input)
+                            setInput('')
+                            setIsFocused(false)
                         }
-                        return;
-                    }
-
-                    // Enter key sends the message, but we still want to allow 
-                    // shift + enter to add a new line.
-                    if (e.key === 'Enter' && !e.shiftKey) {
-                        e.preventDefault();
-                        adjustHeight(true)
-                        onSave(input)
-                        setInput('')
-                    }
-                    // Escape key cancels editing
-                    if (e.key === 'Escape') {
-                        e.preventDefault();
-                        if (onCancel) {
-                            onCancel();
+                        // Escape key cancels editing
+                        if (e.key === 'Escape') {
+                            e.preventDefault();
+                            if (onCancel) {
+                                onCancel();
+                            }
                         }
-                    }
-                }}
-            />
+                    }}
+                />
+                {isDropdownVisible && isFocused && (
+                    <ChatDropdown
+                        options={expandedVariables}
+                        onSelect={handleOptionSelect}
+                        filterText={dropdownFilter}
+                        position={showDropdownAbove ? "above" : "below"}
```

Comment:
Since ChatInput is now always fixed to the bottom of the page, I think we can simplify this file by removing the logic for rendering the variable dropdown. It should always be above. Thoughts?